app/uk/gov/hmrc/agentclientrelationships/controllers/RelationshipsController.scala

```diff
@@ -334,4 +334,20 @@ class RelationshipsController @Inject()(
         )
     }
   }
+
+  def hasLegacyMapping(arn: Arn, nino: Nino): Action[AnyContent] = Action async { implicit request =>
+    implicit val auditData: AuditData = new AuditData()
+    auditData.set("arn", arn)
+    auditData.set("Journey", "hasLegacyMapping")
+    auditData.set("service", "mtd-it")
+    auditData.set("clientId", nino)
+    auditData.set("clientIdType", "nino")
+
```

Comment:
are we sending an event here?

Reply:
lookupCesaForOldRelationship sends an audit event, and takes a implicit auditData as a param